appveyor-windows.yml

```diff
@@ -0,0 +1,85 @@
+version: 1.0.{build}
+image: Windows Server 2019
+build: off
+
+# Change the clone folder to somewhere in "D:\" because this is shared by default with Docker. We need this to mount folders.
+clone_folder: D:\source
+
+environment:
+  AWS_DEFAULT_REGION: us-east-1
+  SAM_CLI_DEV: 1
+
+  # In Windows, tempdir is usually in C:\. But in AppVeyor only D:\ is shared in Docker. 
+  # Therefore change TEMPDIR it to D: for invoke tests to work. Python uses $TMPDIR envvar
+  # to find root of tempdir
+  TMPDIR: D:\tmp
+
+  # MSI Installers only use Py3.6.6. It is sufficient to test with this version here.
+  PYTHON_HOME: "C:\\Python36-x64"
```

Comment:
This doesn't really select Py3 in Windows Server 2019 image. Need to install Py3 separately.